api/src/main/resources/workbench.yaml

```diff
@@ -2191,6 +2211,46 @@ definitions:
         type: integer
         format: int64
         description: Timestamp when the user completed era commons linking.
+      eraCommonsBypassTime:
+        type: integer
+        format: int64
+        description: Timestamp when the user was bypassed for completing era commons linking.
+      complianceTrainingCompletionTime:
```

Comment:
opt: Alternatively we could implement this with a standard structure to avoid some duplication, e.g.:

```
eraCommonsAccess:
  ref: "$/WorkbenchAccess"

WorkbenchAccess:
  properties:
    completionTime:
    bypassTime:
```

I don't feel very strongly either way. It does add a few more potential null-ness states to worry about.